Standards/scs-0114-v1-secure-connections.md

```diff
@@ -0,0 +1,172 @@
+  - If using RabbitMQ, the OpenStack services' oslo.messaging configuration for RabbitMQ MUST use the `kombu_ssl_*` options accordingly to enable SSL.
+  - If using Qpid, the OpenStack services' oslo.messaging configuration for Qpid MUST set the `qpid_protocol` option to `ssl` to enable SSL.
+- If using Apache Kafka, the server listener MUST be configured to accept SSL connections. See [Apache Kafka Listener Configuration](https://kafka.apache.org/documentation/#listener_configuration).
+  - The OpenStack services' oslo.messaging configuration for Kafka MUST specify `security_protocol` as either `SSL` or `SASL_SSL` and the related options appropriately. See [Kafka Driver Options](https://docs.openstack.org/oslo.messaging/latest/admin/kafka.html#driver-options).
+
+### Hypervisor and Live Migration Connections
+
+- If using libvirt on compute nodes, the libvirt port (as per `listen_addr`) MUST NOT be exposed to the network in an unauthenticated and unprotected fashion. It SHOULD be bound to `127.0.0.1`.
+- If QEMU and libvirt are used as the hypervisor interface in Nova, QEMU-native TLS SHOULD be used. See [Secure live migration with QEMU-native TLS](https://docs.openstack.org/nova/latest/admin/secure-live-migration-with-qemu-native-tls.html).
+
+
+### External VM Connections
+
+- As an OPTIONAL measure to assist customers in protecting external connections to their OpenStack networks and VMs, the Neutron VPNaaS service MAY be integrated into the infrastructure and the Neutron VPNaaS API extension offered to users. See the [Neutron VPNaaS documentation](https://docs.openstack.org/neutron-vpnaas/latest/).
+
+### Internal Neutron Connections
+
+- As an OPTIONAL measure to protect Neutron SDN traffic between physical nodes within the infrastructure, IPsec tunnels MAY be established between nodes involved in Neutron networks, such as compute and network controller nodes, at the network interfaces configured in Neutron.
+
+## Related Documents
+
+- [OpenStack Security Guide](https://docs.openstack.org/security-guide/)
+  - [OpenStack Security Guide: Secure communication](https://docs.openstack.org/security-guide/secure-communication.html)
+  - [OpenStack Security Guide: Database transport security](https://docs.openstack.org/security-guide/databases/database-transport-security.html)
+  - [OpenStack Security Guide: Messaging transport security](https://docs.openstack.org/security-guide/messaging/security.html#messaging-transport-security)
+- [Nova Documentation: Secure live migration with QEMU-native TLS](https://docs.openstack.org/nova/latest/admin/secure-live-migration-with-qemu-native-tls.html)
+- [Guide to TLS Standards Compliance](https://www.ssl.com/guide/tls-standards-compliance/)
+
+## Conformance Tests
+
+Conformance tests are limited to communication channels exposed to users, such as the public API interfaces.
+
+<!-- TODO: implement public API tests checking the TLS handshakes including cipher suites etc. -->
```

Comment:
delete this comment please

Reply:
Did this right after I implemented and pushed the test suite to the branch :)